src/envo/envo-edit.owl

```diff
@@ -44974,6 +44976,27 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605005> "epixylon"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605005> <http://purl.obolibrary.org/obo/ENVO_03605000>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605006> (freshwater stream)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_03605006> "A stream where fresh water flows.")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#created_by> <http://purl.obolibrary.org/obo/ENVO_03605006> <ORCID:0000-0001-9076-6066>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#creation_date> <http://purl.obolibrary.org/obo/ENVO_03605006> "2024-01-10T19:07:00Z")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasOBONamespace> <http://purl.obolibrary.org/obo/ENVO_03605006> "ENVO")
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/ENVO_03605006> "ENVO:03605006")
+AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/ENVO_03605006> "Preliminary definition.")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_03605006> "freshwater stream"@en)
+EquivalentClasses(<http://purl.obolibrary.org/obo/ENVO_03605006> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_00000023> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> <http://purl.obolibrary.org/obo/ENVO_00002011>)))
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_03605006> <http://purl.obolibrary.org/obo/ENVO_00000023>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_03605007> (freshwater stream biome)
```

Comment:
More or less same comments as for the other class proposed in this PR.

Reply:
Thanks @pbuttigieg and @cmungall 

This PR was based on patterns that are widespread in EnvO. I agree with you changes and will make them now. I'm thinking about what I can do to help these old patterns, either though documentation or cleanup. Please share your thoughts.